.github/workflows/benchmark.yml

```diff
@@ -166,6 +166,10 @@ jobs:
     #   run: time HALF=1 STEPS=10 BS=1536 GPUS=6 python3 examples/hlb_cifar10.py
     - name: Run full CIFAR training
       run: time HSA=1 HALF=1 LATEWINO=1 STEPS=1000 TARGET_EVAL_ACC_PCT=93 python3 examples/hlb_cifar10.py | tee train_cifar_one_gpu.txt
+    - name: Run 10 MLPerf ResNet50 training steps (1 gpu)
+      run: HSA=1 BENCHMARK=10 BS=104 GPUS=1 MODEL=resnet python3 examples/mlperf/model_train.py | tee train_resnet_one_gpu.txt
```

Comment:
the `BENCHMARK` specified steps per epoch, but it still runs multiple epochs

Reply:
returning early seems to not clean up dataloader properly (same if the process crashed or keyboard interrupt early)

Reply:
what doesn't get cleaned up when return?